general/1_terminal/Yaroslav-Dzhur/hw3.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+
+DOM=$(date +"%d")
+
+LANG=C DOW=$(date +"%a")
```

Comment:
Why do we `LANG=C`?

Reply:
DOW returns day of week in the same language as installed on the computer. I used LANG=C to run the program correctly on computers with different localizations.